examples/02-modal-harmonic/00-multi_harmonic.py

```diff
@@ -3,12 +3,13 @@
 
 Multi-harmonic response example
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 This example shows how to compute a multi-harmonic response
 using ``fft`` transformations.
 
 .. note::
     This example requires the Premium ServerContext.
-    For more information, see :ref:`_ref_getting_started_contexts`.
+    For more information, see :ref:`_user_guide_server_context`.
```

Comment:
```suggestion
    For more information, see :ref:`user_guide_server_context`.
```

Reply:
I think this should be changed everywhere in your PR

Reply:
Done!